staging/src/k8s.io/cloud-provider/config/types.go

```diff
@@ -89,3 +92,12 @@ type CloudProviderConfiguration struct {
 	// cloudConfigFile is the path to the cloud provider configuration file.
 	CloudConfigFile string
 }
+
+type WebhookConfiguration struct {
+	// Webhooks is the list of webhooks to enable or disable
+	// '*' means "all enabled by default webhooks"
```

Comment:
nit: you documented "all webhooks" and "disabled by default webhooks" in `AddFlag`. Is "enabled by default webhooks" the diff of those two groups?

Reply:
Yes, that is correct